doc/ExpertOptions.md

```diff
@@ -1441,13 +1441,25 @@ For an in depth discussion, please see [Missing Call Arguments](MissingCallArgum
 
 # Manifests
 
-In extreme cases, running compiled workflows can fail due to DNAnexus platform limits on the total size of the input and output JSON documents of a job. An example is a task with many inputs/outputs that is called in scatter over a large collection. In such a case, you can enable manifest support at compile time with the `-useManifests` option. This option causes each generated applet or workflow to accept inputs as a manifest, and to produce outputs as a manifest.
-
-A manifest is a JSON document that contains all the inputs/outputs that would otherwise be passed directly to/from the applet. A manifest can be specified in one of two ways: via a JSON input, or via a File input (where the file must exist on the platform).
+In extreme cases, running compiled workflows can fail due to DNAnexus platform limits on the total size of the input and 
+output JSON documents of a job. An example is a task with many inputs/outputs that is called in scatter over a large collection. 
+In such a case, you can enable manifest support at compile time with the `-useManifests` option. 
+This option causes each generated applet or workflow to accept inputs as a manifest, and to produce outputs as a manifest.
+
+A manifest is a JSON document that contains all the inputs/outputs that would otherwise be passed directly to/from the 
+workflow stage. A manifest can be specified in one of two ways: 
+1. A `.json` input file (see [Manifest JSON](#manifest-json)) is the recommended way to provide inputs in the manifest format.  
+`java -jar dxCompiler.jar -inputs mymanifest.json` will produce `mymanifest.dx.json` that can be passed to `dx run`.
+This is a recommended way to provide inputs in the manifest format.
+2. A platform `file-xxx` with content described in  [Intermediate manifest file inputs/outputs](#intermediate-manifest-file-inputs/outputs) 
+section can be used to pass manifest output from a stage of one workflow (including the `output` stage) as input to another workflow.  A 
+typical use case for this scenario is when a user wants to pass manifest output file from a stage (including `output` stage) 
+directly to a new workflow. Also, this scenario might be useful when debugging individual stages of a failing workflow.
 
 ## Manifest JSON
 
-When manifest support is enabled, each applet has an `input_mainfest___` input field of type `hash`, which means that it accepts a JSON document as a string. For example, given the following workflow:
+When manifest support is enabled, each applet has an `input_mainfest___` input field of type `hash`, which means that it 
+accepts a JSON document as a string or as a `.json` file. For example, given the following workflow:
```

Comment:
Sure, but I see no contradiction here. This paragraph (introduced before this ticket) only means that all created applets have `input_mainfest___` in the input spec instead of actual inputs. This paragraph does not touch upon how the inputs are actually passed to those applets when they are executed.